### PR TITLE
Set tags only if passed any.

### PR DESCRIPTION
**Pull request by Fisiu**
Signed-off-by: Mariusz Fik <mariusz@fidano.pl>

### CODE REVIEW

**Comment by nWidart** [comment]
Hi,

Looks like the build is failing. 
I also don't like how you intended everything, there are better ways to do this by for instance inverting the condition.

File: Modules/Tag/Traits/TaggableTrait.php [hunk 1]
```diff
@@ -97,6 +97,11 @@ public static function allTags()
      */
     public function setTags($tags, $type = 'slug')
     {
+        // Avoid null $tags
```
Comment (nWidart) [inline Modules/Tag/Traits/TaggableTrait.php]: Can you remove this comment please? :)

Comment (Fisiu) [reply Modules/Tag/Traits/TaggableTrait.php]: Done. PR updated.

**Comment by nWidart** [comment]
Thanks! 👍

**Pull request merged**